setup-gcloud/src/setup-gcloud.ts

```diff
@@ -62,15 +62,17 @@ async function run(): Promise<void> {
       }
 
       const credsPath = path.join(workspace, uuidv4());
+      const serviceAccountKeyObj = setupGcloud.parseServiceAccountKey(
+        serviceAccountKey,
+      );
       await fs.writeFile(
         credsPath,
-        JSON.stringify(
-          setupGcloud.parseServiceAccountKey(serviceAccountKey),
-          null,
-          2,
-        ), // Print to file as string w/ indents
+        JSON.stringify(serviceAccountKeyObj, null, 2), // Print to file as string w/ indents
       );
-
+      core.exportVariable(
+        'GCLOUD_PROJECT',
```

Comment:
We may want to consider exporting `GOOGLE_CLOUD_PROJECT` since that seems to be standard https://github.com/googleapis/nodejs-common/issues/20.

Reply:
The goal of exporting the ADC and GCLOUD_PROJECT is to the ability to auto authenticate client libraries. It may be useful to also export GOOGLE_CLOUD_PROJECT, but we should think of the potential repercussions first.